model/src/main/java/oracle/kubernetes/weblogic/domain/v2/AdminServer.java

```diff
@@ -26,6 +26,9 @@
   @Description("T3 network access points to export")
   private Map<String, ExportedNetworkAccessPoint> exportedNetworkAccessPoints = new HashMap<>();
 
+  @Description("")
```

Comment:
Perhaps a description like, "Configures which of the admin server's WebLogic admin channels should be exposed outside the Kubernetes cluster via a node port service."